### ISSUE TITLE
Aetherochemical Research Facility - Second boss

### ISSUE THREAD

**Issue by davlar12** [ffxivcode/AutoDuty#158]
So, running with trust as a warrior (this might matter), the second boss of the dungeon has this ability that deals near fatal damage if the target doesn't have a single extra party member in a circle with them. It's then followed by a stack mechanic.

The mod logic seems to keep the player character inside the aoe circle and proceeds to murder the trusts that try to pair up for the mechanic. Seeing as I was running as warrior it manages to survive barely the encounter (though not always).

I'm not sure if its due to another plugin but figured I'd post this here first.

https://github.com/user-attachments/assets/e29f83d2-007b-4d1c-b335-0e96b9f870ae

Video attached to show the mechanic that is being failed. (Ballistic Missile)

I believe the correct way to succeed the mechanic would be to just let the trusts deal with it since you need exactly 2 target to survive it.

This also led me to see a potential failure state later down the line if you do the dungeon with a job capable of somehow surviving the boss (I was on warrior as you can see from the video) since it seems that the ARF configuration assumes you die on this boss.

Surviving the boss leads you to get stuck in an elevator not far after the boss since it skips the reviving portion of the steps.

**Comment by davlar12** [comment]
Tested again with a dps class this time.

The problem seems to be very much specific to using a tank. As dps, the trust behave differently and the player acts as the second target if not the mechanic's target or a single trust comes to the player if the player is the target so it all resolves correctly.

I think I'm wrong as to why the following failure state in the elevator happens but I did see it happen more than once. I'll have to pay attention more.
